mirai-core-api/src/commonMain/kotlin/data/FriendGroup.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019-2022 Mamoe Technologies and contributors.
+ *
+ * 此源代码的使用受 GNU AFFERO GENERAL PUBLIC LICENSE version 3 许可证的约束, 可以在以下链接找到该许可证.
+ * Use of this source code is governed by the GNU AGPLv3 license that can be found through the following link.
+ *
+ * https://github.com/mamoe/mirai/blob/dev/LICENSE
+ */
+
+package net.mamoe.mirai.data
+
+import net.mamoe.mirai.contact.Friend
+
+/**
+ * 一个好友分组.
+ * 可能同时存在多个相同[名称][name]的分组, 但是每个分组的 [id] 都是唯一的.
+ * @see FriendGroups
+ * @since 2.13
+ */
+public interface FriendGroup {
```

Comment:
正文介绍结束后的 `@` 应该隔一行空行, 其他的 review 一样

```suggestion
/**
 * 一个好友分组.
 * 可能同时存在多个相同[名称][name]的分组, 但是每个分组的 [id] 都是唯一的.
 *
 * @see FriendGroups
 * @since 2.13
 */
public interface FriendGroup {
```